tests/unit/test.program.js

```diff
@@ -307,6 +307,19 @@ describe('program.main', () => {
                      process.cwd() + path.sep);
       });
   });
+  it('passes the path of a firefox binary when specified', () => {
```

Comment:
nit: this line should have one line of whitespace above it to show that `it()` is a new function separate from the function above it